packages/adblocker/src/filters/network.ts

```diff
@@ -451,6 +451,83 @@ function getFilterOptionValue(line: string, pos: number, end: number): [number,
   return [pos, value];
 }
 
+/**
+ * https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Regular_expressions/Character_escape
+ * \f, \n, \r, \t, \v, \0, \^, \$, \\, \., \*, \+, \?, \(, \), \[, \], \{, \}, \|, \/
+ * https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Regular_expressions/Character_class_escape
+ * \d, \D, \s, \S, \w, \W
+ * https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Regular_expressions/Word_boundary_assertion
+ * \b, \B
+ */
+const REGEXP_CHARACTER_ESCAPES = new Set([
+  102, 110, 114, 116, 118, 48, 94, 36, 92, 46, 42, 43, 63, 40, 41, 91, 93, 123, 125, 124, 47, 100,
+  68, 119, 87, 115, 83, 98, 66,
+]);
+
+function isHexLiteral(code: number) {
+  return (code >= 48 && code <= 57) || (code <= 65 && code <= 70) || (code >= 97 && code <= 102);
+}
+
+/**
+ * Checks if the character(s) after the escape character should be preserved.
+ */
+function isCharacterEscapeInRegExp(line: string, pos: number, _end: number): [number, boolean] {
+  const code = line.charCodeAt(pos + 1);
+  if (REGEXP_CHARACTER_ESCAPES.has(code) || code === 44 /* ',' */) {
```

Comment:
Probably will not matter much, but I think, there is no downside here to start with the cheaper operation.

```suggestion
  if (code === 44 /* ',' */ || REGEXP_CHARACTER_ESCAPES.has(code)) {
```